proto/osmosis/incentives/gauge.proto

```diff
@@ -41,4 +41,45 @@ message LockableDurationsInfo {
     (gogoproto.stdduration) = true,
     (gogoproto.moretags) = "yaml:\"lockable_durations\""
   ];
+}
+
+message HistoricalReward {
+  // Reward per share for each denom
+  repeated cosmos.base.v1beta1.DecCoin cumulative_reward_ratio = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.DecCoins"
+  ];
+  uint64 period = 2;
+  uint64 last_eligible_epoch = 3;
+}
+
+message CurrentReward {
+  uint64 period = 1;
+  int64 last_processed_epoch = 2;
```

Comment:
Its not obvious to me on read here, why is there a separate period and last_processed_epoch.

Why don't we assume that theres just one period per epoch?

Reply:
(So concretely, only making there be a period variable, and commenting that at the moment its expected to be once per epoch, but in the future this could change)

Or is that what the code already does, and this last_processed_epoch is for debugging or unbonding?